tests/framework/installer/ceph_installer.go

```diff
@@ -235,6 +235,122 @@ func (h *CephInstaller) CreateK8sRookClusterWithHostPathAndDevices(namespace, sy
 	return err
 }
 
+// CreateK8sRookExternalCluster creates rook external cluster via kubectl
+func (h *CephInstaller) CreateK8sRookExternalCluster(namespace, firstClusterNamespace string) error {
+
+	dataDirHostPath, err := h.initTestDir(namespace)
+	if err != nil {
+		return fmt.Errorf("failed to create test dir. %+v", err)
+	}
+	logger.Infof("Creating external cluster: namespace=%q, firstClusterNamespace=%q", namespace, firstClusterNamespace)
+
+	logger.Infof("Creating namespace %s", namespace)
+	ns := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}
+	_, err = h.k8shelper.Clientset.CoreV1().Namespaces().Create(ns)
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return fmt.Errorf("failed to create namespace %q. %v", namespace, err)
+	}
+
+	// Skip this step since the helm chart already includes the roles and bindings
+	if !h.useHelm {
+		logger.Infof("Creating external cluster roles")
+		roles := h.Manifests.GetClusterExternalRoles(namespace, firstClusterNamespace)
+		if _, err := h.k8shelper.KubectlWithStdin(roles, createFromStdinArgs...); err != nil {
+			return fmt.Errorf("failed to create cluster roles. %v", err)
+		}
+	}
+
+	// Inject connection information from the first cluster
+	logger.Info("Injecting cluster connection information")
+	err = h.InjectRookExternalClusterInfo(namespace, firstClusterNamespace)
+	if err != nil {
+		return fmt.Errorf("failed to inject cluster information into the external cluster. %v", err)
+	}
+
+	// Start the external cluster
+	logger.Infof("Starting Rook External Cluster with yaml")
+	settings := &ClusterExternalSettings{namespace, dataDirHostPath}
+	rookCluster := h.Manifests.GetRookExternalCluster(settings)
+	if _, err := h.k8shelper.KubectlWithStdin(rookCluster, createFromStdinArgs...); err != nil {
+		return fmt.Errorf("failed to create rook external cluster. %v ", err)
+	}
+
+	logger.Infof("Rook external cluster started")
```

Comment:
Can we validate that the operator was able to successfully connect to the cluster? This could be done by looking at the status on the cluster CR, right? Otherwise we don't know if the external cluster is properly connected for the test. 

Ideally we would also create a storage class and a client pod that verifies we can read/write to the external cluster.  A separate PR would be good for this so we can get this test re-enabled.

Reply:
I thought this https://github.com/leseb/rook/blob/03705c61f212ba59a4afee8c6181d496860cdeab/tests/integration/ceph_multi_cluster_test.go#L97 would do. wdyt?

Reply:
Right, that looks good for the first point, I missed that.